airflow/config_templates/config.yml

```
@@ -1294,6 +1294,14 @@
       type: string
       example: ~
       default: "True"
+    - name: allow_trigger_in_future
+      description: |
+        Allow externally triggered DagRuns for Execution Dates in the future
+        Only has effect if schedule_interval not set in DAG
```

```suggestion
        Only has effect if schedule_interval is set to None in DAG
```
The default value for `schedule_interval` is actually `timedelta(days=1)` so I thought `is set to None` is a more accurate description of this requirement here.

---

Good Point @yuqian90